api/lib/models/RSVP/RSVP.ts

```diff
@@ -0,0 +1,36 @@
+import jsonAssetLoader from '../../assets/schemas/json-asset-loader';
+import { UidType } from '../../JSCommon/common-types';
+import BaseObject from '../BaseObject';
+
+const rsvpSchema = jsonAssetLoader('rsvpSchema');
+export const TABLE_NAME = 'RSVP';
+
+export interface IRSVPModel {
+  uid?: UidType | null;
```

Comment:
should not have type definition null. `null` is different than `undefined` and causes weird issues.
```suggestion
  uid?: UidType;
```
^^ this alone should be good enough.